roles/passenger/.ansible-lint

```diff
@@ -0,0 +1,15 @@
+exclude_paths:
+  - ./roles/extra_path/.travis.yml
+skip_list:
+  - ANSIBLE0002
+  - ANSIBLE0006
+  - ANSIBLE0010
+  - '503'
+  - '305'
+  - '303'
+  - '306'
+  - '301'
+  - '602'
+  - '204'
```

Comment:
this seems like a lot of stuff...I've disabled one rule so far because it didn't make sense in the context of a test, but we might want to reexamine this at some point and see if we need to be ignoring everything.

Reply:
this was copied from PUL, I haven't looked at these yet actually

important to review before merging? (this gets to your point about avoiding unnecessary tech debt)

Reply:
if we can take the time to figure these out i'd like to do it now...looking at a few of them, some are simple formatting-related things (`ANSIBLE0001` is for trailing whitespace), but others like `ANSIBLE0006` are explicit warnings to avoid using particular modules/patterns. that seems like the kind of debt we want to avoid.

the other thing is that specifying them using only the codes like this is also deprecated (i saw a warning about it today); evidently you're supposed to use [rule names from this list](https://ansible-lint.readthedocs.io/en/latest/default_rules.html) instead, which is at least much nicer to read.

could we start by removing all of these and seeing what `molecule lint` fails on, and go from there? then we could at least re-add the rules using their semantic names, if we need to keep them.